tools/scripts/goimports/main.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// defaultArgs are always passed to goimports.
+// -w: write result to (source) file instead of stdout
+// -local: put imports beginning with this string after 3rd-party packages (comma-separated list)
+// (see: goimports -h)
+var defaultArgs = []string{"-w", "-local", "github.com/pokt-network/poktroll"}
+
+func main() {
+	root := "."
+	var filesToProcess []string
+
+	// Walk the file system and accumulate matching files
+	err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
```

Comment:
OPTIONAL NIT: assign the function to a param above and then just do `err := filepath.Walk(roo, walkFn)` to make it easier to read.